system_files/shared/usr/lib/systemd/system/sb-key-notify.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service to check for secure boot key enrollment and send notifications
+
+[Service]
+ExecStart=/usr/libexec/sb-key-notify.sh
+
+[Install]
+WantedBy=multi-user.target
+
+[Timer]
```

Comment:
`[Timer]` is invalid in a service unit.

You'll need to create a separate timer unit with the Timer settings which references this service if you want it to run on a timer, but I don't know why this needs to run on a timer. Once at boot to find the state should be enough.

Reply:
ya youre right about the timer.  i can edit it to run at first boot instead

Reply:
> ya youre right about the timer. i can edit it to run at first boot instead

it already runs at first boot.